### PR TITLE
Feature : Improve active drop banner behaviour 

### PR DESCRIPTION
**Pull request by domw30**
Link to associated zer0 card:  [Notion Card](https://www.notion.so/Improve-active-drop-banner-behaviour-bc674470f6cb4693b5a2d1d203b977de?p=b1ccaeee918b40cd9fa3c5bcfadbf6a5)

-----

## 1. Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] notion card has been moved to the Review column
- [x] notion card has a link to this PR
- [x] A reviewer has been tagged in the notion card

## 2. PR type

Please check the type of change your PR introduces:
- [x] Feature

## 3. What is the current behaviour?
- The banner moves out of view on scroll. 

## 4. What is the new behaviour?
- The banner is fixed when scroll down motion is detected
- Add smaller banner (message banner)

### CODE REVIEW

File: src/containers/other/PageHeader/PageHeader.module.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	z-index: 10;
+
+	transition: opacity 0.4s ease-in-out, 0.4s ease-in-out;
+	transform: translateY(0);
+}
+
+.BannerContainer {
+	position: fixed;
+	z-index: 3;
+	width: 1234px;
```
Comment (domw30) [inline src/containers/other/PageHeader/PageHeader.module.scss:21]: This should be updated to be responsive

File: src/containers/other/PageHeader/PageHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+				<WilderIcon />
+				{/* Replace children with TitleBar */}
+				{children}
+			</nav>
+			<div
+				className={`${styles.BannerContainer} ${
+					hideNavBar ? styles.ScrollDown : styles.ScrollUp
+				}`}
+			>
+				<WheelsRaffle />
```
Comment (domw30) [inline src/containers/other/PageHeader/PageHeader.tsx:42]: Are other banners used in place of wheels banner? i.e. is this different to cribs banner

File: src/pages/ZNS/ZNS.tsx [hunk 3]
```diff
@@ -489,22 +488,18 @@ const ZNS: React.FC<ZNSProps> = ({ domain, version, isNftView: nftView }) => {
...
 					opacity: hasLoaded ? 1 : 0,
 					transition: 'opacity 0.2s ease-in-out',
-					paddingTop: 145,
+					paddingTop: 271,
 				}}
 			>
-				{/* Nav Bar */}
-				{/* TODO: Make a more generic Nav component and nest FilterBar and TitleBar */}
-				<FilterBar
+				<PageHeader
```
Comment (domw30) [inline src/pages/ZNS/ZNS.tsx:494]: My thinking behind having a PageHeader container here was that the ZNS page file could over time be refactored to look something like:

```
<PageHeader>
    <NavBar />
    <Banner />
</PageHeader>
<PageContent>
    <SomeContent />
    <SomeContent />
</PageContent>
```

File: src/components/Banners/MessageBanner/MessageBanner.module.scss [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+	padding-bottom: 0;
+	color: rgba(255, 255, 255, 1);
+}
+
+.Countdown {
+	font-weight: 700;
+	font-size: 14px;
+	margin-right: 8px;
+	padding-bottom: 0;
+	color: rgba(255, 255, 255, 1);
```
Comment (padala) [inline src/components/Banners/MessageBanner/MessageBanner.module.scss:62]: Please use scss variables available. If required please add new variables.

File: src/components/WilderIcon/WilderIcon.module.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+	top: 0;
+	left: -19px;
+	height: 56px;
+	width: 56px;
+
+	display: flex;
+	justify-content: center;
+	align-items: center;
+
+	background: linear-gradient(180deg, #1c4358 0%, #2a0047 100%);
```
Comment (padala) [inline src/components/WilderIcon/WilderIcon.module.scss:12]: Please use scss variables available. If required please add new variables.

File: src/components/WilderIcon/WilderIcon.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+import styles from './WilderIcon.module.scss';
+
+//- Asset Imports
+import wilderIcon from './assets/WWLogo_SVG.svg';
+
+const WilderIcon: React.FC = () => {
+	//- Hooks
+	const history = useHistory();
+
+	const home = () => {
```
Comment (padala) [inline src/components/WilderIcon/WilderIcon.tsx:15]: rename it to `redirectToHome` or `navigateToHome`

**Review by padala — CHANGES_REQUESTED** [review]

**Review by padala — APPROVED** [review]

**Pull request merged**